LedRGB.cpp

```diff
@@ -35,3 +35,9 @@ void LedRGB::turnOff() {
     greenLed.turnOff();
     blueLed.turnOff();
 }
+
+void LedRGB::blink(unsigned n, unsigned delay1, unsigned delay2) {
+    redLed.blink(n, delay1, delay2);
```

Comment:
This will not work. The blinks will happen consecutively after another. eg  the red will blink then the blue will blink then the green will blink.